IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcGeographicCRS/DocEntity.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGeographicCRS" UniqueId="2d6be663-8d5e-42cc-b5b4-13bb37f41262" BaseDefinition="IfcCoordinateReferenceSystem" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="GeodeticDatum" UniqueId="b4d581f7-612a-4e98-963a-95ad06714397" DefinedType="IfcIdentifier" AttributeFlags="1" />
+		<DocAttribute Name="PrimeMeridian" UniqueId="b2c9cf50-b4bb-4979-92d8-c44a05f22954" DefinedType="IfcIdentifier" AttributeFlags="1">
+			<Documentation>The identification of the meridian defining zero longitude in the used geographic CRS.</Documentation>
+			<Definition>
+				<DocDefinitionRef xsi:nil="true" href="IfcIdentifier_1dG8O0DIbDBxufYev6dUDH" />
+			</Definition>
+		</DocAttribute>
+		<DocAttribute Name="Unit" UniqueId="82bf71dc-4f74-483b-84c7-70c84a5ce22d" DefinedType="IfcNamedUnit" AttributeFlags="1" />
```

Comment:
Have been thinking about the *height above the ellipsoid* - would it require a separate unit as well? If yes, another WHERE rule ensuring it is a Length Unit would also be needed.

Reply:
I read this as referencing the documentation text: "Any location on Earth can be referenced by a point with longitude and latitude coordinates and the **height above the ellipsoid**". I would guess that no action needed (same unit for all coordinates).

Reply:
The problem @pjanck is pointing out is that `First` and `SecondCoordinate` can be either `IfcLenghtMeasure` or `IfcPlaneAngleMeasure`. However, Height is already restricted so no need to do anything there.

Please look at `IfcRigidOperation`. The type there is still `IfcPositivePlaneAngleMeasure`.

Reply:
> The problem @pjanck is pointing out is that `First` and `SecondCoordinate` can be either `IfcLenghtMeasure` or `IfcPlaneAngleMeasure`. However, Height is already restricted so no need to do anything there.

I wan't refereing to that. Rather, I was addressing

> the project units would apply

I would like to *allow* (similarly to how `IfcProjectedCRS` defines a Unit, or how for First and SecondCoordinate there is a Unit) for a different height-unit of the `IfcGeographicCRS`, compared to the project units. This way, a project using feet could still be located x meters above/below an ellipsoid (using `IfcRigidOperation`).